### DISCUSSION TITLE
RDS database links

### DISCUSSION THREAD

**Discussion by tekumara** [acorn-io/runtime#598] [General]
Are there any examples of linking to an RDS database?

**Comment by cjellick** [comment]
We don’t have an example, but that would be great to have in the docs. We’ll add it.

Here’s the deal: there is a k8s native way to do this using a service with a external name. This random blog article does a good job explaining it: https://medium.com/@ManagedKube/kubernetes-access-external-services-e4fd643e5097


Create a service like that in the same namespace as the App resource (the acorn namespace by default) and then you can reference it by name via a link, like —link myexternaldb:db


We plan on improving this in the future by introducing an acorn “externalService” type so that you don’t have to create the k8s service.
